src/app/core/store/collections/collection.util.ts

```diff
@@ -17,10 +17,20 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Attribute} from '../../../../core/dto/attribute';
+import {CollectionModel} from './collection.model';
 
-export interface ConfiguredAttribute extends Attribute {
+export function getDefaultAttributeId(collection: CollectionModel): string {
+  if (collection.defaultAttributeId) {
+    const attributeExist = collection.attributes.findIndex(attr => attr.id === collection.defaultAttributeId) !== -1;
+    if (attributeExist) {
+      return collection.defaultAttributeId;
```

Comment:
What about this?
```typescript
return collection.attributes.find(attr => attr.id === collection.defaultAttributeId);
```

Reply:
No, because I want to return value only if `defaultAttributeId` is valid.

Reply:
Oh, I see.